adapters/projects/robinos/prediction.go

```diff
@@ -0,0 +1,117 @@
+package robinos
+
+import (
+	"context"
+	"math/big"
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/taikoxyz/trailblazer-adapters/adapters"
+	"github.com/taikoxyz/trailblazer-adapters/adapters/contracts/robinos"
+)
+
+const (
+	RobinosAddress                = "0x9EF9c57754eD079D750016b802DcCD45d0AB66f8"
+	LogRewardDistributedSignature = "RewardDistributed(string,uint256[],address[])"
+)
+
+var SelectedMultiplierEvents = []string{"Prediction - $TAIKO price on Oct 20", "Serie A 24/25 - Juventus v. Lazio", "EPL 24/25 - Liverpool v. Chelsea", "EPL 24/25 - Man Utd v. Brentford"}
+
+type PredictionIndexer struct {
+	client    *ethclient.Client
+	addresses []common.Address
```

Comment:
given that the events will be updated weekly, i recommend adding
events []string as part of the struct
`
events    []string
`